deep_speech_2/README.md

```diff
@@ -459,10 +459,11 @@ Mandarin | [Internal Mandarin Model](to-be-added) | Baidu Mandarin Dataset | 291
 
 #### Language Model Released
 
-Language Model | Training Data | Token-based | Size | Filter Configuraiton
-:-------------:| :------------:| :-----: | -----: | -----------------:
-[English LM](http://paddlepaddle.bj.bcebos.com/model_zoo/speech/common_crawl_00.prune01111.trie.klm) |  To Be Added | Word-based | 8.3 GB | To Be Added
-[Mandarin LM](http://cloud.dlnel.org/filepub/?uuid=d21861e4-4ed6-45bb-ad8e-ae417a43195e) |  To Be Added | Character-based | 2.8 GB | To Be Added
+Language Model | Training Data | Token-based | Size | Description
+:-------------:| :------------:| :-----: | -----: | :-----------------:
+[English LM](http://paddlepaddle.bj.bcebos.com/model_zoo/speech/common_crawl_00.prune01111.trie.klm) |  [en.00.deduped.xz](http://web-language-models.s3-website-us-east-1.amazonaws.com/ngrams/en/deduped/en.00.deduped.xz) | Word-based | 8.3 GB | pruned with 0 1 1 1 1 <br/> about 1.85 billion n-grams <br/> 'trie'  binary with '-a 22 -q 8 -b 8'
```

Comment:
1. “Large internal data" --> "Internal Data"
2. "pruned ..." --> "Pruned ... ;"
3. "about ... " --> "About ...;"
4. left-alignment for the last column 
5. "en.00.deduped.xz" --> "CommonCrawl(en.00)"

Reply:
Description --> Descriptions

Reply:
Done.